unix/qstrdefsport.h

```diff
@@ -79,6 +79,7 @@ Q(listen)
 Q(accept)
 Q(recv)
 Q(recvfrom)
+Q(settimeout)
```

Comment:
Not true.  All qstrdef*.h files go through the C preprocessor first, then through makeqstrdata.py.  So you can use #if's.  See line 153 of py/py.mk.  (Maybe this is not true though for windows build?)